src/index.ts

```diff
@@ -1,6 +1,23 @@
-import type { AxiosError, AxiosRequestConfig, AxiosInstance, AxiosStatic } from 'axios';
+import type {
+  AxiosError,
+  AxiosRequestConfig,
+  AxiosInstance,
+  AxiosStatic,
+  AxiosInterceptorManager,
+  AxiosResponse,
+  InternalAxiosRequestConfig
+} from 'axios';
 import isRetryAllowed from 'is-retry-allowed';
 
+interface AxiosResponseInterceptorManagerExtended extends AxiosInterceptorManager<AxiosResponse> {
+  handlers: Array<{
+    fulfilled: ((value: AxiosResponse) => AxiosResponse | Promise<AxiosResponse>) | null;
+    rejected: ((error: any) => any) | null;
+    synchronous: boolean;
+    runWhen: (config: InternalAxiosRequestConfig) => boolean | null;
+  }>;
```

Comment:
The following JavaScrip implementations have been reviewed and typedefs defined.
https://github.com/axios/axios/blob/v1.6.2/lib/core/InterceptorManager.js#L23